src/main/java/com/github/gokolo/personaldashboard/dao/AddressDAOImpl.java

```diff
@@ -55,6 +57,24 @@ public AddressDTO save(final AddressDTO address) {
     }
 
     @Override
+    public List<AddressDTO> findAll() {
+        Statement statement;
```

Comment:
Tip:
There is no need to declare every local object at the top of your method.
In this particular case, you could spare 2 lines if you declare and create the `Statement` and `ResultSet` objects in the same line (inside the `try`).